tests/model_test.py

```diff
@@ -0,0 +1,26 @@
+"""Tests for the Post model."""
+
+import pytest
```

Comment:
The task is to use the built-in test capabilities of Django and avoid using pytest at this point.

Reply:
I'll use Django's TestCase class.

Reply:
Marking as resolved.